lib/index.ts

```diff
@@ -386,6 +392,40 @@ async function setup(
 		},
 	);
 
+	app.post(
+		'/dut/flashFromFile',
+		jsonParser,
+		async (req: express.Request, res: express.Response) => {
+			res.writeHead(202, {
+				'Content-Type': 'text/event-stream',
+				Connection: 'keep-alive',
+			});
+
+			const timer = setInterval(() => {
+				res.write('status: pending');
+			}, 5000);
+
+			
+			try {
+				let FILENAME = req.body.filename;
+				if(FILENAME.includes(`.gz`)){
+					console.log(`Unzipping file`)
+					console.log(await execSync(`gunzip -f ${FILENAME}`))
+					FILENAME = FILENAME.replace(/\.gz$/, '');
+				}
+				console.log(`Attempting to flash with file: ${FILENAME}...`);
+				await worker.flash(FILENAME);
```

Comment:
```suggestion
				await worker.flash(FILENAME.replace(/\.gz$/, ''));
```

Are we deleting the zipped file after the unzipping. Just making sure the clean up happens.